### PR TITLE
add route constants; enable direct method page

### PR DESCRIPTION
**Pull request by YingXue**
image

### CODE REVIEW

File: src/localization/locales/en.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
             "interfaces": "Interface",
             "events": "Telemetry",
             "eventsPerInterface": "Telemetry",
-            "methods": "Device Methods",
+            "methods": "Direct Method",
```
Comment (andrew-buckley) [inline src/localization/locales/en.json]: Wonder if this should be changed to "Direct methods" to follow the capitalization of the other menu items.

Comment (andrew-buckley) [reply src/localization/locales/en.json]: Please check elsewhere too

File: src/app/devices/deviceContent/components/deviceMethods/deviceMethods.tsx [hunk 2]
```diff
@@ -45,119 +44,133 @@ export default class DeviceMethods extends React.Component<DeviceMethodsProps &
...
 
+    private readonly showCommandBar = (context: LocalizationContextInterface) => {
+        return (
+            <CommandBar
+                className="command"
+                items={[
+                    {
+                        ariaLabel: context.t(ResourceKeys.deviceMethods.invokeMethodButtonText),
+                        disabled: !this.formReady(),
+                        iconProps: {iconName: REFRESH},
```
Comment (andrew-buckley) [inline src/app/devices/deviceContent/components/deviceMethods/deviceMethods.tsx]: Is there a better icon for this?

File: src/localization/locales/en.json (hunk 1 above)
Comment (YingXue) [reply src/localization/locales/en.json]: Good catch! I was only paying attention to the portal...

File: src/app/devices/deviceContent/components/deviceMethods/__snapshots__/deviceMethods.spec.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,95 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`deviceMethods matches snapshot 1`] = `
+<Fragment>
+  <StyledCommandBarBase
+    className="command"
+    items={
+      Array [
+        Object {
+          "ariaLabel": "deviceMethods.invokeMethodButtonText",
```
Comment (andrew-buckley) [inline src/app/devices/deviceContent/components/deviceMethods/__snapshots__/deviceMethods.spec.tsx.snap]: Should this change to `directMethods` to match the changes elsewhere?

**Comment by andrew-buckley** [comment]
I see a few discrepancies in the usage of direct vs device method throughout the code. I'd just have a scan through and see where things should be changed. I also wonder if it should be `Direct methods` rather than `Direct method` to match the portal.

File: src/app/devices/deviceContent/components/deviceMethods/deviceMethods.tsx (hunk 2 above)
Comment (YingXue) [reply src/app/devices/deviceContent/components/deviceMethods/deviceMethods.tsx]: Maybe we can use what portal use

**Comment by YingXue** [comment]
> I see a few discrepancies in the usage of direct vs device method throughout the code. I'd just have a scan through and see where things should be changed. I also wonder if it should be `Direct methods` rather than `Direct method` to match the portal.

Portal uses 'Direct Method'. So I will be using 'Direct method' here to match this tool's convention.

**Pull request merged**